keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/ExternalSchemaModelConverter.kt

```diff
@@ -0,0 +1,82 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.swagger.v3.core.converter.AnnotatedType
+import io.swagger.v3.core.converter.ModelConverter
+import io.swagger.v3.core.converter.ModelConverterContext
+import io.swagger.v3.oas.models.media.ComposedSchema
+import io.swagger.v3.oas.models.media.Schema
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+
+@Component
+class ExternalSchemaModelConverter : BaseModelConverter() {
```

Comment:
This can load a `Schema` object from `src/main/resources` then merge it with the schema that's been auto-generated. Useful for adding external documentation or overriding specific things.